config/app.config

```diff
@@ -3,6 +3,7 @@
 ---
 
 db_connection_string: 'sqlite:///my.db'
+alembic_path: 'alembic/'
```

Comment:
It will end up under the current working directory. So I think you need to make sure the `db_connection_string` gets set to something sensible when deploying.